db/install.xml

```diff
@@ -19,6 +19,8 @@
         <FIELD NAME="collapsesubchapters" TYPE="int" LENGTH="2" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
         <FIELD NAME="grade" TYPE="int" LENGTH="3" NOTNULL="true" DEFAULT="0" SEQUENCE="false" COMMENT="grade"/>
         <FIELD NAME="printing" TYPE="int" LENGTH="2" NOTNULL="true" DEFAULT="0" SEQUENCE="false" COMMENT="stores printing status"/>
+        <FIELD NAME="peersharing" TYPE="int" LENGTH="2" NOTNULL="false" DEFAULT="1" SEQUENCE="false" COMMENT="Students can share contributions with other students"/>
+        <FIELD NAME="timeofday" TYPE="int" LENGTH="2" NOTNULL="false" DEFAULT="0" SEQUENCE="false" COMMENT="Dates show hour/minute of the day"/>
```

Comment:
These need adding to backup/moodle2/backup_giportfolio_stepslib.php (assuming they are settings that should be retained across backup & restore).

Reply:
Fields added in backup now